src/zabapgit_objects_impl.prog.abap

```diff
@@ -232,6 +232,9 @@ CLASS lcl_objects IMPLEMENTATION.
 
 
     lv_length = strlen( iv_start ).
+    IF lv_length > strlen( iv_path ).
+      lcx_exception=>raise( 'Error: folder structure' ).
+    ENDIF.
```

Comment:
I'd suggest a more clear message ... smth like `Start folder incompatible with package path {...}`
BTW maybe it is better to compare content too ? some kind of startsWith() ? I mean '/src' and '/zmypackage' will pass the condition. Or I'm messing something ?